addons/remass/BI.hpp

```diff
@@ -0,0 +1,97 @@
+    // 556 STANAG Magazines (Includes ACE Magazines)
+    class 30Rnd_556x45_Stanag: CA_Magazine {
+        mass = 10;
+    };
+    class 30Rnd_556x45_Stanag_green: 30Rnd_556x45_Stanag {};
```

Comment:
None those should be necessary unless one of them specifically sets it in their own config (instead of inheriting). And if that's the case, should overwrite specifically here as well otherwise it won't work. Thus my conclusion is they inherit properly and this is not needed here.

And if that is true you can get rid of some dependencies. 

Same for the ones below.